ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHDashboardSettings.resource

```diff
@@ -343,3 +382,267 @@ Set RHODS Users Group To Inexistent Group
+    Wait Until Page Contains Element    ${DELETE_ACCELERATOR_PROFILE_BTN}
+    Click Button    ${DELETE_ACCELERATOR_PROFILE_BTN}
+    Wait Until Page Contains Element    ${ACCELERATOR_DELETE_MODAL_INPUT}
+    Input Text    ${ACCELERATOR_DELETE_MODAL_INPUT}    ${name}
+    Click Button    ${ACCELERATOR_DELETE_MODAL_DELETE_BUTTON}
+
+Create An Accelerator Profile Via CLI
+    [Documentation]    Create an instance of Accelerator Profile using OC
+    [Arguments]   ${name}  ${ns}=redhat-ods-applications
+    ${rc}    ${out}    Run And Return Rc And Output   oc create -f ${ACCELERATOR_RESOURCES_DIRPATH}/${name}.yaml -n ${ns}
+    Should Be Equal As Integers    ${rc}     0
+
+Create An Accelerator Profile Via UI
+    [Documentation]    Fill Accelerator Profile values with the ones provided in the arguments and click on
+    ...                create button
+    ...                view: Create Accelerator profile
+    [Arguments]   ${name}  ${identifier}  ${description}=${EMPTY}  ${enabled}=True
+    ...           ${tolerations}=${EMPTY}  ${tol_operator}=Equal  ${tol_effect}=None  ${tol_key}=key
+    ...           ${tol_value}=value  ${tol_seconds}=Forever
+    Wait Until Page Contains Element  ${ACCELERATOR_NAME}  timeout=5s
+    Input Text    ${ACCELERATOR_NAME}    ${name}
+    Input Text    ${ACCELERATOR_IDENTIFIER}    ${identifier}
+    IF    "${description}" != "${EMPTY}"
+        Input Text    ${ACCELERATOR_DESCRIPTION}    ${description}
+    END
+    # By default are enabled
+    IF  ${enabled} != True
+        Click Element    ${ACCELERATOR_ENABLED_SWITCH}
+    END
+    # Tolerations
+    IF    "${tolerations}" != "${EMPTY}"
+        Click Button    ${ACCELERATOR_ADD_TOLERATION_BTN}
+        Wait Until Element Is Visible    ${ACCELERATOR_ADD_TOLERATION_MODAL_FORM}
+        # Select Operator
+        Click Button    ${ACCELERATOR_TOLERATION_OPERATOR_DROPDOWN}
+        Click Element    //div[@data-testid="toleration-operator-select"]//div[text()="${tol_operator}"]
+        # Select Effect
+        Click Button    ${ACCELERATOR_TOLERATION_EFFECT_DROPDOWN}
+        Click Element    //label[@for="effect-select"]/parent::*/following::div//*[text()="${tol_effect}"]
+        # Input Key and value
+        Input Text    ${ACCELERATOR_TOLERATION_KEY}    ${tol_key}
+        Input Text    ${ACCELERATOR_TOLERATION_VALUE}    ${tol_value}
+        # Toleration Seconds
+        IF    "${tol_seconds}" != "Forever"
+            Click Element    ${ACCELERATOR_TOLERATION_SECONDS_CUSTOM_RADIO}
+            Clear element and input text    ${ACCELERATOR_TOLERATION_SECONDS_CUSTOM_VALUE}   ${tol_seconds}
+        ELSE
+            Click Element    ${ACCELERATOR_TOLERATION_SECONDS_FOREVER_RADIO}
+        END
+        Click Button    ${ACCELERATOR_TOLERATION_ADD_OR_UPDATE_BUTTON}
+    END
+    Click Button    ${CREATE_OR_UPDATE_ACCELERATOR_PROFILE_DETAIL_BTN}
+
+Modify The Accelerator Profile
```

Comment:
could you call it "Edit Accelerator Profile" for consistency with similar keywords?